tools/demo/publisher.py

```diff
@@ -46,7 +49,7 @@ def send_message(msg, subject, registry, publisher):
         password=RABBITMQ_PASSWORD,
         vhost=RABBITMQ_VHOST,
     )
-    publisher = BasicPublisher(rabbitmq_details)
+    publisher = BasicPublisher(rabbitmq_details, publish_retry_delay=5, publish_max_retries=36, verify_cert=False)
```

Comment:
Was this intended to use the constants in tol_lab_share/config/defaults.py ?

Reply:
Yes but only for this script in particular. It is a demo about how to publish to the queues, so all default configuration comes from defaults.py and any specific settings from environment variables